patches/minecraft/net/minecraft/entity/passive/HorseArmorType.java.patch

```diff
@@ -0,0 +1,61 @@
+--- ../src-base/minecraft/net/minecraft/entity/passive/HorseArmorType.java
++++ ../src-work/minecraft/net/minecraft/entity/passive/HorseArmorType.java
+@@ -37,7 +37,6 @@
+         return this.ordinal();
+     }
+ 
+-    @SideOnly(Side.CLIENT)
```

Comment:
Since you are not using `getHash()` any more, you should leave the `@SideOnly(Side.CLIENT)` there, and consider deprecating it.